### DISCUSSION TITLE
brew install git-lfs@3.5.1 Warning: No available formula with the name "git-lfs@3.5.1". Did you mean git-lfs?

### DISCUSSION THREAD

**Discussion by devCracker** [git-lfs/git-lfs#5740] [Q&A]
Why all of sudden  git-lfs@x.x.x formula is not working in home-brew while installing.  
Getting this below warning and not installing. 

Warning: No available formula with the name "git-lfs@3.5.1". Did you mean git-lfs?

It is happening since yesterday evening

brew install git-lfs@3.5.1
Warning: No available formula with the name "git-lfs@3.5.1". Did you mean git-lfs?
==> Searching for similarly named formulae and casks...
==> Formulae
git-lfs ✔

To install git-lfs ✔, run:
  brew install git-lfs ✔

**Comment by bk2204** [comment]
Hey,

We don't control the Git LFS Homebrew formula, but sometimes there are additional versions of formulae with an `@` sign, such as `gcc@13`.  Typically, these are older versions which are required for compatibility or which users may want to continue to use as a dependency.

It seems unlikely that Homebrew would have named a formula `git-lfs@3.5.1`, since that's the latest version, but you'd have to ask them.  I see that the `git-lfs` formula was updated to 3.5.1 two months ago, though.
